lib/parallel_tests/tasks.rb

```diff
@@ -251,7 +251,7 @@ def rails_61_or_greater?
       executable = File.expand_path('../../bin/parallel_test', __dir__)
 
       command = [*ParallelTests.with_ruby_binary(executable), type, '--type', test_framework]
-      command += ['-n', count] if count
+      command += ['-n', count.to_s] if count
       command += ['--pattern', pattern] if pattern
       command += ['--test-options', options] if options
       command << pass_through if pass_through
```

Comment:
I feel like this would also be wrong since you could only pass through a single command ...

Reply:
@jdufresne do you think https://github.com/grosser/parallel_tests/commit/5ac27e9f551840722c830cf72bec64a3e2ef325d broke passthrough ?

Reply:
yeah ... 
```
      expect(ParallelTests::Tasks.parse_args(args)).to eq([2, "plain", "-p default --group-by steps", "--runtime-log /path/to/log"])
```
so this needs a shellsplit and concat